### ISSUE TITLE
Bug 59184466: [React Native Gallery Preview]: User is unable to access the 'A Horizontal Scrollview' scroll bar using keyboard.

### ISSUE THREAD

**Issue by protikbiswas100** [microsoft/react-native-gallery#651]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Keyboard

Repro-Steps:
Launch the React Native Gallery-Preview app.
Navigate to the 'Expand menu' button and invoke it.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Scrollview’ button present under “Scrolling” group and invoke it.
Now try to access the 'A Horizontal Scrollview' scroll bar using keyboard and observe the behavior
Actual Result:​
User is unable to access the 'A Horizontal Scrollview' scroll bar using keyboard

Note: User is able to access the 'A Horizontal Scrollview' scroll bar button using mouse

Expected Result:
'A Horizontal Scrollview' scroll bar should be accessible using keyboard.

User Impact:
Users who rely on keyboard might feel difficult to access 'A Horizontal Scrollview' scroll bar.

MAS Reference Link: [MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)

Trap ID: [2.1 - Physical Challenge](https://aka.ms/PhysicalChallenge)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
